.github/workflows/android-perf.yml

```diff
@@ -171,9 +129,7 @@ jobs:
     needs: set-parameters
     secrets: inherit
     strategy:
-      matrix:
-          model: ${{ fromJson(needs.set-parameters.outputs.models) }}
-          delegate: ${{ fromJson(needs.set-parameters.outputs.delegates) }}
+      matrix: ${{ fromJson(needs.set-parameters.outputs.benchmark_configs) }}
```

Comment:
@huydhn do you know how I can customize the job title? It looks like it's automatically generated as part of the `matrix:` configuration. Currently it's displayed as "[export-models (meta-llama/Llama-3.2-1B, llama3_htp, arn:aws:devicefarm:us-west-2:308535385114:dev... / linux-job](https://github.com/pytorch/executorch/actions/runs/12401578767/job/34621284740#logs)", where the arn is not useful, I'd like to replace it with the human readable device pool name.

Reply:
AFAIK, there is no way to change this behavior from GitHub.  The name always include all the matrix dimensions.  Maybe there is a way to create a more human-friendly alias for the pool on AWS, let me take a look from that angle

Reply:
Don't worry. No need to waste time on this. I can always include the device_name in the matrix

Reply:
Added "device_name" and "device_arn" to the matrix, and put the "device_arn" at the end. Now we can see each job is titled with device pool name